consensus/hotstuff/votecollector/combined_vote_processor_v3.go

```diff
@@ -31,7 +31,7 @@ import (
 // the proposer's vote (decorator pattern).
 // nolint:unused
 type combinedVoteProcessorFactoryBaseV3 struct {
-	committee   hotstuff.Committee
+	committee   hotstuff.Replicas
```

Comment:
same argument as for [combined_vote_processor_v2.go](https://github.com/onflow/flow-go/blob/4de4231e43a65fe672a54aab4f84971ecac03db6/consensus/hotstuff/votecollector/combined_vote_processor_v2.go), I think we should use the `DynamicCommittee` here to reject votes from ejected replicas. 

## ⚠️ 
if you follow my suggestion, the following line needs to be updated:
https://github.com/onflow/flow-go/blob/4de4231e43a65fe672a54aab4f84971ecac03db6/consensus/hotstuff/votecollector/combined_vote_processor_v3.go#L89